packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -1,41 +1,38 @@
 :host {
-  @apply bg-foreground-1
-    text-color-3
-    box-border
-    inline-block
-    select-none
-    duration-150
-    ease-in-out;
+  background-color: var(--calcite-color-foreground-1);
```

Comment:
Can we revert the Tailwind ➡️ CSS changes? I'd like to keep TW utils until we wrap up https://github.com/Esri/calcite-design-system/issues/6602 (mainly for visual regression testing purposes).